libs/tokens/src/hooks/tokens/unsupported/useAreUnsupportedTokens.ts

```diff
@@ -0,0 +1,15 @@
+import { useCallback } from 'react'
+import { useIsUnsupportedToken } from './useIsUnsupportedToken'
+
+type NullishAddress = string | null | undefined
```

Comment:
I assume you don't want to use `Nullish` because that's in the main app and it would create a circular dependency?

Reply:
You know my mind :)

Reply:
I would like to move common types to some a new lib in the future